src/asm/fstack.c

```diff
@@ -204,6 +230,8 @@ bool fstk_FindFile(char const *path, char **fullPath, size_t *size)
 	return false;
 }
 
+static void tryNextIncludeFile(void); // forward declaration for yywrap
```

Comment:
I wanted to keep the actual definition of `tryNextIncludeFile` next to `fstk_RunInclude` since they're similar.